function_calling/README.md

```diff
@@ -9,6 +9,24 @@
 Function Calling kit
 ======================
 
+<!-- TOC -->
```

Comment:
In the app image, could you set the icons of the tools in another colour? In red, it looks like they are "wrong".
Maybe cornflower blue, or the same orange as SambaNova. I vote for cornflower blue.

Reply:
I agree, added streamlit/config.toml (in some cases user should select in stremlit menu ->Settings -> theme -> custom theme)